src/content/pages/pyladies-events.mdx

```diff
@@ -20,14 +24,21 @@ WHERE?
 Prague Congress Centre (PCC), Open Space 2 (Rooms 223 + 224)
 
 ## PyLadies Lunch
-Join us for a special lunch event aimed at fostering community and empowerment in tech. Enjoy meaningful conversations and networking opportunities. Thank you to our sponsor 🐙 Kraken Tech 🐙 for supporting the lunch event.
+Join us for a special lunch event aimed at fostering community and empowerment in tech. Enjoy meaningful conversations and networking opportunities.
 
 WHEN?
 Thursday, 11th July 2024, 12:30 to 14:00.
 
 WHERE?
 Prague Congress Centre (PCC), VIP lunch area.
 
+<a href="https://kraken.tech/" target="_blank" rel="noopener noreferrer">
+    <Image src={krakenLogo} width="400" alt="Kraken Logo" />
+    <div style={{ marginBottom: '-30px' }}> </div>
```

Comment:
```suggestion
    <Image src={krakenLogo} width="400" alt="Kraken Logo" class="-mb-30" />
```

Reply:
Thank you @egeakman! 

Actually I tried `class="-mb-30"`  initially, but it rendered this, which has more spacing than I wanted: 
![image](https://github.com/EuroPython/website/assets/6319134/ce130ae5-84a4-49cf-9c56-c9a6fce0154a)
 
 <div style={{ marginBottom: '-30px' }}> </div> gives me the spacing I preferred:  
 
![image](https://github.com/EuroPython/website/assets/6319134/4db39131-6fdd-4580-8dae-d8c68a01cde9)

Do you know why?

Reply:
Hmm, I don't know why. Astro assets normally accept classes, maybe it doesn't work well with Tailwind?

I can't look into this further right now. I think it's perfectly OK to merge it as is if it works!

Thanks again @hypha 🤗

Reply:
Yeah I also thought it is not worth taking up your time to look into this now. We can talk in Prague! I will merge with thanks, @egeakman!